checkman/aix_lsvg_all

```diff
@@ -0,0 +1,13 @@
+title: Mirror state of all LVM logical volumes on AIX
```

Comment:
checkman/ is for manpages of check plugins. You can put the documentation of inventory plugins in the module docstrings, for now.